src/features/sidebarSuite/subComponents/settings/DbSourceFilters/styledComponents.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box, styled } from "@mui/material";
+
+export const InputOutlineBox = styled(Box)(({ theme }) => ({
+  borderRadius: theme.shape.borderRadius,
+  border: `1px solid ${theme.palette.grey[400]}`,
+  minHeight: "4rem",
+  padding: theme.spacing(1),
+}));
+
+export const ComboboxBox = styled(Box)(({ theme }) => ({
```

Comment:
Combobox Box! I like it :)

Reply:
:rofl: This is just the magic I bring! :sparkles: